engine/security/trust/index.md

```diff
@@ -104,7 +113,7 @@ read how to [manage keys for DCT](trust_key_mng.md).
 
 Within the Docker CLI we can sign and push a container image with the
 `$ docker trust` command syntax. This is built on top of the Notary feature
-set, more information on Notary can be found [here](/notary/getting_started/).
+set. For more information, see the [Notary](https://github.com/theupdateframework/notary){:target="_blank" rel="noopener" class="_"} GitHub repository.
```

Comment:
Perhaps change the link caption to `Notary GitHub repository` (easier to click, and more descriptive for the link)

```suggestion
set. For more information, see the [Notary GitHub repository](https://github.com/theupdateframework/notary){:target="_blank" rel="noopener" class="_"}.
```